services/monitor/instances/monitor_gci_group_settings/constraints/to_be_adapted/constraint.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: CIGroupSettingsConstraintV1
+metadata:
+  name: to_be_adapted
+  annotations:
+    description: All non mailing lists groups must conform to these settings.
+    category: Identity and Access Management
+spec:
+  severity: major
+  match:
+    target: ['directories/'] # to_be_adapted
```

Comment:
Why between single quotes?
For organizations it works without quotes

Reply:
corrected